tests/all/js_globals/WebAssembly.rs

```diff
@@ -0,0 +1,30 @@
+#![allow(non_snake_case)]
+
+use super::project;
+
+#[test]
+fn validate() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
+
+            extern crate wasm_bindgen;
+            use JsValue;
+            use wasm_bindgen::prelude::*;
+            use wasm_bindgen::js::WebAssembly;
+
+            #[wasm_bindgen]
+            pub fn validate_wasm(wasm: JsValue) -> bool {
+                WebAssembly::validate(wasm).unwrap_or(false)
```

Comment:
Hmm. This will "swallow" a possible error, so you can't really test for that case. Could you take a look at how methods that possibly throw are tested (eg. `tests/all/js_globals/Number.rs`). I am thinking about something like:
```rust
match WebAssembly::validate(wasm) {
    Ok(value) => value,
    Err(_err) => "TypeError".into(), // additionally, you can check the contents of the '_err' - I am not sure now, but I think it should contain the "TypeError" string
}
```

Reply:
In this case, the output type is `bool` or `String`, and I'm not sure how to type that :-).

Reply:
You're right - sorry for that confusion! I wonder whether simply returning a result from that method would work for both success and error cases on js side (so that the error could be caught within `try/catch` or `assert.throws`).

Reply:
...or, we can try a testing strategy similar to that of `decode_uri_component` in `test/all/js_globals/mod.rs` 🙂

Reply:
Btw. the contents of _err are the actual javascript error message.
Istead of `Err(_err) => "TypeError".into()` you can just do `Err(_err) => _err`.